Library/Formula/citus.rb

```diff
@@ -0,0 +1,55 @@
+class Citus < Formula
+  desc "PostgreSQL-based distributed RDBMS"
+  homepage "https://www.citusdata.com"
+  url "https://github.com/citusdata/citus/archive/v5.0.0.tar.gz"
+  sha256 "a72bd7e9020c11f19d08e58f1f8aa8e83e7f1f377facb6c8020fcaa917f9a3ee"
+
+  head "https://github.com/citusdata/citus.git"
+
+  depends_on "postgresql"
+
+  def install
+    config_args = %W[--prefix=#{prefix} PG_CONFIG=#{Formula["postgresql"].opt_bin}/pg_config]
+
+    # workaround for https://github.com/Homebrew/homebrew/issues/49948
+    make_args = ["libpq=-L#{Formula["postgresql"].opt_lib} -lpq"]
+
+    system "./configure", *config_args
```

Comment:
Done. In fact, the `configure` argument was entirely unnecessary, so I removed it.